incubator/jaeger/values.yaml

```diff
@@ -0,0 +1,79 @@
+# Default values for jaeger.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
```

Comment:
We usually group by component instead of resource type. I think that would make more sense.

Reply:
Looking to refactor for this now.

Reply:
Refactored grouping by component.